pointgrey_camera_driver/debian/postinst

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+#DEBHELPER#
+
+# A udev rules file was added during install, load and trigger rules
+if [ "$1" = configure ]; then
+  if which udevadm > /dev/null; then
+    set +e # Disable exit on error
+    udevadm control --reload-rules
+    service udev restart
+    udevadm trigger
+    set -e # Re-enable exit on error
+  fi
+fi
```

Comment:
Okay, I would concur that `dh_installudev` doesn't actually take care of triggering udevadm.

```
$ wget http://packages.ros.org/ros/ubuntu/pool/main/r/ros-indigo-pointgrey-camera-driver/ros-indigo-pointgrey-camera-driver_0.12.2-0trusty-20170313-103752-0700_amd64.deb
$ dpkg-deb -R ros-indigo-pointgrey-camera-driver_0.12.2-0trusty-20170313-103752-0700_amd64.deb tmp
$ cat tmp/DEBIAN/postinst
#!/bin/sh
set -e
# Automatically added by dh_installudev
if [ "$1" = configure ]; then
	if [ -e "/etc/udev/rules.d/z60_ros-indigo-pointgrey-camera-driver.rules" ]; then
		echo "Preserving user changes to /etc/udev/rules.d/40-ros-indigo-pointgrey-camera-driver.rules ..."
		if [ -e "/etc/udev/rules.d/40-ros-indigo-pointgrey-camera-driver.rules" ]; then
			mv -f "/etc/udev/rules.d/40-ros-indigo-pointgrey-camera-driver.rules" "/etc/udev/rules.d/40-ros-indigo-pointgrey-camera-driver.rules.dpkg-new"
		fi
		mv -f "/etc/udev/rules.d/z60_ros-indigo-pointgrey-camera-driver.rules" "/etc/udev/rules.d/40-ros-indigo-pointgrey-camera-driver.rules"
	fi
fi
# End automatically added section
# Automatically added by dh_makeshlibs
if [ "$1" = "configure" ]; then
	ldconfig
fi
# End automatically added section
```

I see some discussion about this here, wrt to why `dh_installudev` doesn't automatically trigger: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=697362

I would rather not complicate this package unnecessarily; IMO replugging the device or manually running `udevadm trigger` after installation is an okay compromise.